### PR TITLE
Added Cobertura config

### PR DESCRIPTION
**Pull request by dracco1993**
## Problem

Turns out, we're not _actually_ reporting code coverage.

## Solution

Add cobertura to do such things.

## Testing Notes

Check to see if Code Climate is reporting code coverage.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,8 +12,10 @@ before_script:
   - ./cc-test-reporter before-build
 script:
   - ./gradlew check
+  - ./gradlew cobertura
```
Comment (randomtestfive) [inline .travis.yml]: do we need to run this?

File: build.gradle [hunk 2]
```diff
@@ -56,6 +59,10 @@ task wrapper(type: Wrapper) {
     gradleVersion = '4.4'
 }
 
+cobertura {
+    coverageFormats = ['html', 'xml']
```
Comment (randomtestfive) [inline build.gradle]: should travis gen html

Comment (dracco1993) [reply build.gradle]: I left this in for local development.  I don't think it _really_ matters either way, but I find it interesting.

File: src/test/java/org/redalert1741/robotbase/input/InputTest.java [hunk 3]
```diff
@@ -36,10 +36,19 @@ void edgeDetectOneChangeTest() {
...
      */
     @Test
-    void edgeDetectAlternateTest() {
+    public void edgeDetectAlternateTest() {
         EdgeDetect edge = new EdgeDetect();
         for(int i = 0; i < 10; i++) {
             assertEquals(i%2!=0, edge.check(i%2!=0));
         }
     }
+
```
Comment (tricktrap) [inline src/test/java/org/redalert1741/robotbase/input/InputTest.java]: Can we get rid of this test? Seems like we don't need it anymore.

**Pull request merged**